lib/exabgp/configuration/setup.py

```diff
@@ -230,6 +230,12 @@
 		},
 	},
 	'api':  {
+		'lines': {
```

Comment:
not sure about the name neither .. 'api', 'grouping' but I can live with 'lines'